tests/test_hep_bd1xx.py

```diff
@@ -1704,6 +1704,27 @@ def test_authors_from_700__a_double_e_handles_multiple_roles():
     assert expected == result['authors']
 
 
+def test_authors_from_100__a_removes_trailing_commas_and_spaces():
+    schema = load_schema('hep')
+    subschema = schema['properties']['authors']
+
+    snippet = (
+        '<datafield tag="100" ind1=" " ind2=" ">'
+        '  <subfield code="a">, Lenske,</subfield>'
```

Comment:
did you invent this or is it real? all tests are supposed to use real data if possible.

Reply:
we hand ```Lenske,``` in the real case that I had found in the exceptions dashboard. But I added the ```, ``` in front because I also found cases like that and I wanted to combine in one test. I'll separate in two different tests then and add the recid